icaruscode/Filters/FilterDataIntegrity_module.cc

```diff
@@ -0,0 +1,96 @@
+////////////////////////////////////////////////////////////////////////
+//
+// FilterDataIntegrity class
+//
+////////////////////////////////////////////////////////////////////////
+#include <fstream>
```

Comment:
Not needed. If you want to include something:
```suggestion
#include <set>
#include <string>
```

Reply:
ditto